lib/sanbase_web/graphql/resolvers/metric_resolver.ex

```diff
@@ -89,6 +93,9 @@ defmodule SanbaseWeb.Graphql.Resolvers.MetricResolver do
          {:ok, result} <- fit_from_datetime(result, args) do
       {:ok, result |> Enum.reject(&is_nil/1)}
     else
+      false ->
+        {:error, "Cannot use current selector: #{selector}. Please input a correct selector"}
```

Comment:
Selector is a map - it does not implement the String.Chars protocol.
 ```suggestion
        {:error, "The provided selector is not valid."}
```